test/framework/cluster.go

```diff
@@ -388,6 +392,15 @@ func (e *ClusterE2ETest) kubeconfigFilePath() string {
 	return filepath.Join(e.ClusterName, fmt.Sprintf("%s-eks-a-cluster.kubeconfig", e.ClusterName))
 }
 
+func (e *ClusterE2ETest) managementKubeconfigFilePath() string {
```

Comment:
Curious, why do we need this?
Ideally `ClusterE2ETest` should stay agnostic from the management vs workload distinction. It's just represents a test over an eksa cluster.

The multi cluster test is the one that should hold those concerns.

Reply:
To validate the workload cluster's CAPI objects in management cluster. It's going to be much more refactoring work and I prefer to do it in separate PR. We can refactor this when we modularize the cluster validation.